config/database.php

```diff
@@ -1,108 +0,0 @@
-<?php
```

Comment:
this file was deleted from the project folder instead of discarding the changes.

please make another commit to this branch that includes this file